### PR TITLE
[BugFix] [seatunnel-engine-server] Incorrect method call for JSON format in AllNodeLogServlet #8217

### PR DESCRIPTION
**Pull request by showfine**
### Purpose of this pull request

Fix an issue where the method `allNodeLogFormatHtml` was incorrectly called for `JSON` log format in the `doGet` method of `AllNodeLogServlet`.

### Does this PR introduce _any_ user-facing change?

No

### How was this patch tested?

There are no unit test cases available, so you can use curl to simulate requesting a jobId for testing.

### Check list

* [x] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [x] If necessary, please update the documentation to describe the new feature. https://github.com/apache/seatunnel/tree/dev/docs
* [x] If you are contributing the connector code, please check that the following files are updated:
  1. Update [plugin-mapping.properties](https://github.com/apache/seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  2. Update the pom file of [seatunnel-dist](https://github.com/apache/seatunnel/blob/dev/seatunnel-dist/pom.xml)
  3. Add ci label in [label-scope-conf](https://github.com/apache/seatunnel/blob/dev/.github/workflows/labeler/label-scope-conf.yml)
  4. Add e2e testcase in [seatunnel-e2e](https://github.com/apache/seatunnel/tree/dev/seatunnel-e2e/seatunnel-connector-v2-e2e/)
  5. Update connector [plugin_config](https://github.com/apache/seatunnel/blob/dev/config/plugin_config)
* [x] Update the [`release-note`](https://github.com/apache/seatunnel/blob/dev/release-note.md).

### CODE REVIEW

File: seatunnel-engine/seatunnel-engine-server/src/main/java/org/apache/seatunnel/engine/server/rest/servlet/AllNodeLogServlet.java [hunk 1]
```diff
@@ -66,7 +66,7 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp)
             FormatType formatType = FormatType.fromString(req.getParameter("format"));
             switch (formatType) {
                 case JSON:
-                    writeJson(resp, logService.allNodeLogFormatHtml(jobId));
+                    writeJson(resp, logService.allNodeLogFormatJson(jobId));
```
Comment (Hisoka-X) [inline seatunnel-engine/seatunnel-engine-server/src/main/java/org/apache/seatunnel/engine/server/rest/servlet/AllNodeLogServlet.java:69]: cc @zhangshenghang  and @liugddx

Comment (Hisoka-X) [reply seatunnel-engine/seatunnel-engine-server/src/main/java/org/apache/seatunnel/engine/server/rest/servlet/AllNodeLogServlet.java:69]: We missed add some test case in https://github.com/apache/seatunnel/blob/06df51bcbbb3294f40ebb5607c70bc031c5175f8/seatunnel-e2e/seatunnel-engine-e2e/connector-seatunnel-e2e-base/src/test/java/org/apache/seatunnel/engine/e2e/joblog/JobLogIT.java#L56. Could you help to add test case @showfine ?

Comment (showfine) [reply seatunnel-engine/seatunnel-engine-server/src/main/java/org/apache/seatunnel/engine/server/rest/servlet/AllNodeLogServlet.java:69]: Sure.

Comment (zhangshenghang) [reply seatunnel-engine/seatunnel-engine-server/src/main/java/org/apache/seatunnel/engine/server/rest/servlet/AllNodeLogServlet.java:69]: > cc @zhangshenghang and @liugddx

Yes, he's right. thks

**Comment by liugddx** [comment]
You can retry the wrong case instead of re-executing all cases

**Comment by Hisoka-X** [comment]
Fixed by #8387

**Pull request closed without merging**